uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/components/leave-type-wizard.component.js

```diff
@@ -189,9 +202,11 @@ define([
     /**
      * Opens the next tab in the active section
      */
-    function openNextActiveSectionTab () {
+    function nextTabHandler () {
       var activeTabIndex = getActiveTabIndex();
+      var activeTab = getActiveTab();
```

Comment:
Do we really need two separate function for `getActiveTab` and `getActiveTabIndex`? can we not merge them in a single function?

Reply:
Yes, I agree, it's ugly. I refactored the code a bit, now we store indexes on the go, so we don't have to loop through sections/tabs every time we want to get the active ones.